main.go

```diff
@@ -176,6 +178,43 @@ func locateDotGit(dir string) (string, error) {
 	return "", fmt.Errorf("Couldn't find .git. Maybe you are not inside of a git repo")
 }
 
+func getURLAliases() (map[string]string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return map[string]string{}, err
+	}
+
+	path := path.Join(usr.HomeDir, ".gitconfig")
+	configArr, err := ioutil.ReadFile(path)
+	if err != nil {
+		return map[string]string{}, err
+	}
+
+	configStr := string(configArr)
+	regex := regexp.MustCompile("\\[url \"([-\\w]+)@(github.com[:]?|gitlab.com[:]?)\"\\]")
+	regexResult := regex.FindAllStringSubmatch(configStr, -1)
```

Comment:
Please use something like https://godoc.org/gopkg.in/go-ini/ini.v1#File.Sections to get the list of all available sections. This approach doesn't look reliable to me.

Reply:
So sorry you had to look at this. I feel so bad I wrote that, and I want to rewind time and don't make this code. Look at new changes and tell me what else should I fix